### PR TITLE
Bug 1822845 update manual image pruning documentation

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1822845

### CODE REVIEW

**Comment by bmcelvee** [comment]
Preview build: http://file.rdu.redhat.com/~bmcelvee/20202307/BZ1822845-4.3/applications/pruning-objects.html#pruning-images_pruning-objects

The notice about exposing the registry is just before the Pruning Images section.

File: modules/pruning-images.adoc [hunk 1]
```diff
@@ -5,42 +5,106 @@
...
 If the integrated registry uses a Redis cache, you must clean the database
 manually.
-
++
 If redeploying the registry after pruning is not an option, then you must
 permanently disable the cache.
-
++
 `oc adm prune images` operations require a route for your registry. Registry
 routes are not created by default.
```
Comment (bmcelvee) [inline modules/pruning-images.adoc:110]: Add + for table

**Comment by bmcelvee** [comment]
@wzheng1 would you mind reviewing this PR, please? Thank you!

**Comment by bmcelvee** [comment]
@xltian would you mind reviewing this PR, please? Thank you!

**Comment by wzheng1** [comment]
@bmcelvee LGTM, thanks!

File: modules/pruning-images.adoc [hunk 2]
```diff
@@ -5,111 +5,152 @@
 [id="pruning-images_{context}"]
 = Pruning images
 
-In order to prune images that are no longer required by the system due to age,
-status, or exceed limits, administrators can run the following command:
+Administrators can manually prune images that are no longer required by the system due to age, status, or exceed limits. There are two methods to manually prune images:
 
+* Running image pruning as a `Job` or `CronJob` on the cluster.
+* Running the `oc adm prune images [<options>]` command.
```
Comment (mburke5678) [inline modules/pruning-images.adoc:11]: Do we need `[<options>]` here?  
_Running the `oc adm prune images` command._ and introduce the options later?

File: modules/pruning-images.adoc [hunk 3]
```diff
@@ -5,111 +5,152 @@
...
+
+.Prerequisites
+
+* To manually prune images, you must first log in to the CLI as a user with an access token. The user must also have the cluster role of `system:image-pruner` or greater (for example, `cluster-admin`).
```
Comment (mburke5678) [inline modules/pruning-images.adoc:15]: s/cluster role of `system:image-pruner` or greater/`system:image-pruner` cluster role or greater ??

Comment (mburke5678) [reply modules/pruning-images.adoc:15]: Do we need _To manually prune images,_ here? 
Seems a bit confusing to me. Sounds like you need this role for one of the two methods, which isn't true(?).

File: modules/pruning-images.adoc [hunk 4]
```diff
@@ -5,111 +5,152 @@
...
+* Expose the image registry.
+
+.Procedure
+
+Running image pruning as a `Job` or `CronJob` on the cluster.
+
+* To manually prune images that are no longer required by the system due to age, status, or exceed limits create a YAML file for the `pruner` service account, for example:
```
Comment (mburke5678) [inline modules/pruning-images.adoc:22]: Do we need _To manually prune images that are no longer required by the system due to age, status, or exceed limits_?
Just _create a YAML file for the `pruner` service account, for example:_ ?

File: modules/pruning-images.adoc [hunk 5]
```diff
@@ -5,111 +5,152 @@
...
+              name: image-pruner
+            serviceAccountName: pruner
 ----
 
-Currently, to prune images you must first log in to the CLI as a user with an
-access token. The user must also have the cluster role `system:image-pruner` or
-greater (for example, `cluster-admin`).
+Running the `oc adm prune images [<options>]` command.
 
+* To manually prune images that are no longer required by the system due to age, status, or exceed limits by running the following command:
```
Comment (mburke5678) [inline modules/pruning-images.adoc:84]: Same here, remove _To manually prune images that are no longer required by the system due to age, status, or exceed limits by_? 
Just: _Run the following command_?

File: modules/pruning-images.adoc [hunk 6]
```diff
@@ -5,111 +5,152 @@
...
-of requests to storage, and increase the speed of processing the request.
-Pruning does not update the integrated registry cache. Images pushed after
-pruning that contain pruned layers will be broken, because the pruned layers
-that have metadata in the cache will not be pushed. Therefore, it is necessary
-to clear the cache after pruning. This can be accomplished by redeploying the
-registry:
-
+imagestreams. Images are non-namespaced resources. Therefore, limiting pruning to a particular namespace makes it impossible to calculate their current usage.
++
+* By default, the integrated registry caches blobs metadata to reduce the number of requests to storage, and increase the speed of processing the request. Pruning does not update the integrated registry cache. Images pushed after pruning that contain pruned layers will be broken, because the pruned layers that have metadata in the cache will not be pushed. Therefore, it is necessary to clear the cache after pruning. This can be accomplished by redeploying the registry:
```
Comment (mburke5678) [inline modules/pruning-images.adoc:96]: Do you know if this applies only to _oc adm prune images_?

Comment (mburke5678) [reply modules/pruning-images.adoc:96]: S/Images pushed after pruning that contain pruned layers/Images that contain pruned layers pushed after pruning ??

File: modules/pruning-images.adoc [hunk 7]
```diff
@@ -5,111 +5,152 @@
...
-
++
 If redeploying the registry after pruning is not an option, then you must
 permanently disable the cache.
-
++
 `oc adm prune images` operations require a route for your registry. Registry
 routes are not created by default.
-
++
```
Comment (mburke5678) [inline modules/pruning-images.adoc:110]: I assume the changes below here are all to remove the carriage returns?

File: modules/pruning-images.adoc [hunk 8]
```diff
@@ -5,111 +5,152 @@
...
 
+* Running image pruning as a `Job` or `CronJob` on the cluster.
+* Running the `oc adm prune images [<options>]` command.
+
+.Prerequisites
+
+* To manually prune images, you must first log in to the CLI as a user with an access token. The user must also have the cluster role of `system:image-pruner` or greater (for example, `cluster-admin`).
+* Expose the image registry.
+
+.Procedure
```
Comment (mburke5678) [inline modules/pruning-images.adoc:18]: Would the procedure work better as something like:
```
.Procedure

To manually prune images that are no longer required by the system due to age, status, or exceed limits, use one of the following methods:

*  Run image pruning as a `Job` or `CronJob` on the cluster by creating a YAML file for the `pruner` service account, for example:
...

* Run the `oc adm prune images [<options>]` command:
+
----
$ oc adm prune images [<options>]
----
```

File: modules/pruning-images.adoc (hunk 4 above)
Comment (mburke5678) [reply modules/pruning-images.adoc:22]: Should you mention that the YAML creates the service account, creates a ClusterRoleBinding for the service account, and creates a cronjob?

File: modules/pruning-images.adoc [hunk 9]
```diff
@@ -5,111 +5,152 @@
...
-
++
 If the integrated registry uses a Redis cache, you must clean the database
 manually.
-
++
 If redeploying the registry after pruning is not an option, then you must
 permanently disable the cache.
-
++
```
Comment (mburke5678) [inline modules/pruning-images.adoc:107]: Should the route  be a prerequisite?

File: modules/pruning-images.adoc (hunk 6 above)
Comment (mburke5678) [reply modules/pruning-images.adoc:96]: s/caches blobs metadata/caches blob metadata ??

Not sure if my suggestion is accurate. Might read better, though.

File: modules/pruning-images.adoc (hunk 2 above)
Comment (bmcelvee) [reply modules/pruning-images.adoc:11]: That makes sense. Thanks!

**Comment by mburke5678** [comment]
@bmcelvee A few things.

File: modules/pruning-images.adoc (hunk 9 above)
Comment (bmcelvee) [reply modules/pruning-images.adoc:114]: I think it's covered by exposing the registry. :) Thanks!

File: modules/pruning-images.adoc (hunk 7 above)
Comment (bmcelvee) [reply modules/pruning-images.adoc:117]: Yep!

**Pull request merged**